torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -367,7 +368,14 @@ std::string dtypesToStr(const std::vector<std::string>& types) {
 
 std::string stacksToStr(const std::vector<std::string>& stacks) {
   std::ostringstream oss;
-  std::copy(stacks.begin(), stacks.end(), std::ostream_iterator<std::string>(oss, ";"));
+  std::transform(
+      stacks.begin(),
+      stacks.end(),
+      std::ostream_iterator<std::string>(oss, ";"),
+      // replace the windows backslash with forward slash
+      [](std::string s) -> std::string {
+        return std::regex_replace(s, std::regex("\\\\"), "/");
```

Comment:
Wouldn't that achieve the same but faster?
```suggestion
        std::replace(s.begin(), s.end(), '\\', '/');
        return s;
```

Reply:
make sense, will change it